$OpenBSD: patch-CMakeLists_txt,v 1.6 2017/05/22 19:38:59 espie Exp $
Use original SQLite driver instead of "tweaked" one, unbreaks runtime.
Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -84,7 +84,7 @@ endif()
 macro(SET_DEFAULT_DB_BACKEND)
     set(_backend ${ARGV0})
     if("${_backend}" STREQUAL "SQLITE")
-        set(AKONADI_DATABASE_BACKEND QSQLITE3)
+        set(AKONADI_DATABASE_BACKEND QSQLITE)
         set(AKONADI_BUILD_QSQLITE TRUE)
     else()
         if("${_backend}" STREQUAL "POSTGRES")
